github-repo-manager/src/main/java/com/axonivy/github/file/GitHubMissingFilesDetector.java

```diff
@@ -102,9 +107,16 @@ private void addMissingFile(GHRepository repo) throws IOException {
     var defaultBranch = repo.getBranch(repo.getDefaultBranch());
     var sha1 = defaultBranch.getSHA1();
     repo.createRef("refs/heads/" + reference.meta().branchName(), sha1);
-    repo.createContent().branch(reference.meta().branchName()).path(reference.meta().filePath()).content(reference.content())
-        .message(reference.meta().commitMessage()).commit();
+    repo.createContent()
+      .branch(reference.meta().branchName())
+      .path(reference.meta().filePath())
+      .content(reference.content())
+      .message(reference.meta().commitMessage())
+      .commit();
     var pr = repo.createPullRequest(reference.meta().pullRequestTitle(), reference.meta().branchName(), repo.getDefaultBranch(), "");
+    if (ghActor != null) {
+      pr.setAssignees(ghActor);
```

Comment:
even if we do an 'auto-merge' the pipeline now documents the running as assignee ... this allows users to negotiate this PR even back in time.